tests/tests_test.go

```diff
@@ -107,7 +107,7 @@ func OpenTestConnection(cfg *gorm.Config) (db *gorm.DB, err error) {
 
 func RunMigrations() {
 	var err error
-	allModels := []interface{}{&User{}, &Account{}, &Pet{}, &Company{}, &Toy{}, &Language{}, &Coupon{}, &CouponProduct{}, &Order{}, &Parent{}, &Child{}, &Tools{}}
+	allModels := []interface{}{&User{}, &Account{}, &Pet{}, &Company{}, &Toy{}, &Language{}, &Coupon{}, &CouponProduct{}, &Order{}, &Parent{}, &Child{}, &Tools{}, &Owner{}, &Building{}, &Furniture{}}
```

Comment:
If it is not a public table, it can be migrated in the test case. e.g. https://github.com/go-gorm/gorm/blob/master/tests/preload_test.go#L337

Reply:
yes, much better thank you